sections/newsletter.liquid

```diff
@@ -1,4 +1,5 @@
 {{ 'component-newsletter.css' | asset_url | stylesheet_tag }}
+{{ 'component-rte.css' | asset_url | stylesheet_tag }}
```

Comment:
Also a duplicate?

Reply:
Good catch! Fixed this [and the other one](https://github.com/Shopify/dawn/pull/2085#discussion_r1114557785) in 7d9f077.